specs/tldr-lint.spec.js

```diff
@@ -1,183 +1,186 @@
+/* eslint-disable no-magic-numbers */
```

Comment:
This is necessary for the tests that use `.toBe(3)` or `.toBe(4)` or whatever in their expect. Given that it's a test file, not much harm just ignoring it altogether.